userspace/libscap/compat/misc.h

```diff
@@ -33,16 +33,12 @@ limitations under the License.
 #define CLOCK_BOOTTIME 7
 #endif
 
-static int sys_bpf(enum bpf_cmd cmd, union bpf_attr *attr, unsigned int size)
```

Comment:
@gnosek - I moved the static inline implementation into scap_bpf.c - the only module using it. 
It caused compilation errors when this file is included, due to "defined but not used".

Reply:
Can we move it to e.g. libscap/compat/bpf.h? I'd rather see compatibility cruft separated from actual code

Reply:
We can. The problem is that it forces anyone including the file to use the function, otherwise the compilation is going to fail, due to declared but not used function.

Reply:
No, I mean moving the sys_bpf/sys_perf_event_open functions to a separate bpf-specific header and including it only where they're needed.

But I guess ideally we'd have two separate headers, with just one function definition each. That sounds like overkill so I'll just stop complaining ;) realistically there's only one place they're used so we might as well define them in scap_bpf.c directly.